espnet2/train/preprocessor.py

```diff
@@ -377,3 +387,103 @@ def __call__(
                 data[text_n] = np.array(text_ints, dtype=np.int64)
         assert check_return_type(data)
         return data
+
+
```

Comment:
CommonPreprocessor can work with only inputting speech. I think SpeechPreprocessor is not necessary.

If you need new preprocessor, could you move the preprocessor class to espnet2/diar/preprocessor.py? (I think this other task should also do it.)

Reply:
Ohh, I didn't aware of that. Thanks for reminding me! Then I think I will keep CommonPreprocessor for now.